src/shared/sleep-config.c

```diff
@@ -90,7 +90,7 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         if (!sc->states[SLEEP_HYBRID_SLEEP])
                 sc->states[SLEEP_HYBRID_SLEEP] = strv_new("disk");
         if (sc->hibernate_delay_sec == 0)
-                sc->hibernate_delay_sec = 2 * USEC_PER_HOUR;
+                sc->hibernate_delay_sec = 1 * USEC_PER_HOUR;
```

Comment:
This seems like it should go in a separate config with a detailed explanation of why we're changing this

Reply:
I don't think it should be a separate setting, it's the same thing as before, it configures how long is the first sleep. We could leave it at 2hr by default so that there's no change at all.

Reply:
Sorry, typo, I meant separate commit

Reply:
right - I think we can leave the old default as-is instead, users can always change it to make it shorter if they want to